soh/src/code/audio_load.c

```diff
@@ -1366,16 +1368,23 @@ void AudioLoad_Init(void* heap, size_t heapSize) {
     int startingSeqNum = MAX_AUTHENTIC_SEQID; // 109 is the highest vanilla sequence
     qsort(customSeqList, customSeqListSize, sizeof(char*), strcmp_sort);
 
+    int seqNum = startingSeqNum;
+
     for (size_t i = startingSeqNum; i < startingSeqNum + customSeqListSize; i++) {
+        // ensure that what would be the next sequence number is actually unassigned in AudioCollection
+        while (AudioCollection_GetSequenceName(seqNum) != NULL) {
```

Comment:
since we're not actually using the sequence name here at all, it probably makes sense to make a new method in `AudioCollection` for checking if a sequence with that key exists in the collection

it should be easy enough to just wrap std::map contains (or std::map count == 0 if cpp 20 ends up being an issue) 
https://en.cppreference.com/w/cpp/container/map/contains
https://en.cppreference.com/w/cpp/container/map/count

Reply:
Yeah, I wasn't sure adding a new function for just one check would be worth it, but it was an easy enough add.